internal/controller/runtime/fsm/runtime_fsm_create_shoot_dry_run.go

```diff
@@ -0,0 +1,40 @@
+package fsm
+
+import (
+	"context"
+	"fmt"
+
+	imv1 "github.com/kyma-project/infrastructure-manager/api/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func sFnCreateShootDryRun(_ context.Context, m *fsm, s *systemState) (stateFn, *ctrl.Result, error) {
+	m.log.Info("Create shoot [dry-run]")
+
+	newShoot, err := convertShoot(&s.instance, m.ConverterConfig)
+	if err != nil {
+		m.log.Error(err, "Failed to convert Runtime instance to shoot object [dry-run]")
+		return updateStatePendingWithErrorAndStop(
+			&s.instance,
+			imv1.ConditionTypeRuntimeProvisioned,
+			imv1.ConditionReasonConversionError,
+			"Runtime conversion error")
+	}
+
+	s.shoot = &newShoot
+	s.instance.UpdateStateReady(
+		imv1.ConditionTypeRuntimeProvisioned,
```

Comment:
if someone filters through type=Provisioned, that person might thing that it was actually done by KIM.

How about having a separate type, such as `ProvisionedDryRun`?